deploy/kustomize/olm-catalog/bases/pmem-csi-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,131 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: pmem-csi-operator.vX.Y.Z
+  namespace: placeholder
+  annotations:
+    categories: Storage
+    containerImage: intel/pmem-csi-driver:vX.Y.Z
+    description: An operator for deploying and managing the PMEM-CSI driver
+    repository: https://github.com/intel/pmem-csi/tree/release-X.Y
+    support: Intel
+    createAt: 2020-06-25 10:50Z
+spec:
+  displayName: Operator for PMEM-CSI driver
+  provider:
+    name: Intel® Corporation
+    url: https://intel.com
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  links:
+  - name: PMEM-CSI
+    url: https://intel.github.io/pmem-csi/X.Y
+  keywords:
+  - PMEM-CSI
+  - PMEM-CSI operator
+  - Persistent Memory
+  - PMEM
+  maintainers:
+  - email: olev.kartau@intel.com
+    name: Olev Kartau
+  - email: patrick.ohly@intel.com
+    name: Patrick Ohly
+  - email: amarnath.valluri@intel.com
+    name: Amarnath Valluri
+  customresourcedefinitions:
```

Comment:
What is this for?

It's not in the generated `deploy/olm-catalog/0.7.0/pmem-csi-operator.clusterserviceversion.yaml`

But perhaps that's because of a failure:
```
$ make operator-generate-catalog VERSION=0.7.0
go: creating new go.mod: module tmp
go: found sigs.k8s.io/controller-tools/cmd/controller-gen in sigs.k8s.io/controller-tools v0.3.0
Generating CRD ...
/nvme/gopath/bin/controller-gen crd:trivialVersions=true,crdVersions=v1beta1 paths=./pkg/apis/... output:dir=deploy/kustomize/olm-catalog/crd/
trap 'make operator-clean-crd' EXIT
make[1]: Entering directory '/nvme/gopath/src/github.com/intel/pmem-csi'
rm -rf deploy/kustomize/olm-catalog/crd
make[1]: Leaving directory '/nvme/gopath/src/github.com/intel/pmem-csi'
Generating base catalog ...
trap "sed -i -e 's;0.7.0;X.Y.Z;g' deploy/kustomize/olm-catalog/kustomization.yaml" EXIT
2020/08/20 14:27:36 absolute path error in '/nvme/gopath/src/github.com/intel/pmem-csi/deploy/kustomize/olm-catalog/crd' : evalsymlink failure on '/nvme/gopath/src/github.com/intel/pmem-csi/deploy/kustomize/olm-catalog/crd' : lstat /nvme/gopath/src/github.com/intel/pmem-csi/deploy/kustomize/olm-catalog/crd: no such file or directory
Error: accumulating resources: accumulating resources from './crd': open /nvme/gopath/src/github.com/intel/pmem-csi/deploy/kustomize/olm-catalog/crd: no such file or directory
Generating package manifests version 0.7.0
Package manifests generated successfully in deploy/olm-catalog
```

Reply:
The error is gone now (commit ccd1b42), but I still don't understand why you are listing this "customresourcedefinitions" here.

Reply:
I listed the `resources` in the wrong place. That should be under `coustomresourcedefintions.owned`. I fixed it now.